security/pkg/nodeagent/test/cert_rotation/rotation_test.go

```diff
@@ -21,63 +21,100 @@ import (
 
 	"istio.io/istio/mixer/test/client/env"
 	sdsTest "istio.io/istio/security/pkg/nodeagent/test"
+	"os/exec"
+)
+
+const (
+	rotateInterval = 1 * time.Second
+	proxyRunningTime = 4 * rotateInterval
+	sleepTime = 100 * time.Millisecond
+	httpSuccessCode = 200
+	certShouldUpdateCount = 2
+)
+
+var (
+	certSet = make(map[string]bool)
```

Comment:
Using `map[string]struct{}` is more idiomatic as there is no confusion if `false` and unset are distinct. Its also more common in the codebase